### PR TITLE
[test][localvolumepath] testing for localvolumepath is now working

### PR DESCRIPTION
**Pull request by alejandroEsc**
This enables testing for localvolumepath addon

### CODE REVIEW

File: test/addons_test.go [hunk 1]
```diff
@@ -24,8 +24,10 @@ import (
 	"github.com/mesosphere/kubeaddons/pkg/test/cluster/kind"
 )
 
-const defaultKubernetesVersion = "1.15.6"
-
+const (
+	defaultKubernetesVersion = "1.15.6"
+	patchStorageClass = "{\"metadata\": {\"annotations\":{\"storageclass.kubernetes.io/is-default-class\":\"false\"}}}"
```
Comment (shaneutt) [inline test/addons_test.go]: Would this be better/cleaner?
```suggestion
	patchStorageClass = `{"metadata":{"annotations":{"storageclass.kubernetes.io/is-default-class":"false"}}}`
```

File: test/addons_test.go [hunk 2]
```diff
@@ -107,6 +114,7 @@ func testgroup(t *testing.T, groupname string) error {
 		return err
 	}
 
+
```
Comment (shaneutt) [inline test/addons_test.go]:
```suggestion
```

File: test/groups.yaml [hunk 3]
```diff
@@ -77,6 +77,18 @@ istio:
...
+# Local Volume Provisioner
+#
+# All Local Volume Provisioner related addons should be tested as a part of this group
+# ------------------------------------------------------------------------------
+localvolumeprovisioner:
+    - "cert-manager"
+    - "defaultstorageclass-protection"
+    - "localvolumeprovisioner"
+
+
```
Comment (shaneutt) [inline test/groups.yaml]:
```suggestion
```

File: test/groups.yaml [hunk 4]
```diff
@@ -77,6 +77,18 @@ istio:
...
+
```
Comment (shaneutt) [inline test/groups.yaml]:
```suggestion
```

File: test/addons_test.go (hunk 1 above)
Comment (alejandroEsc) [reply test/addons_test.go]: Thought i tried that but with json issues. Ill retry, maybe removing the first quotes will work.

**Pull request merged**